label_sync/labels.yaml

```diff
@@ -697,7 +697,122 @@ repos:
         target: both
         addedBy: label
       - color: 0052cc
-        description: Issues or PRs related to code in /prow
+        description: Issues or PRs related to prow
         name: area/prow
         target: both
         addedBy: label
+      - color: 0052cc
+        description: Issues or PRs related to prow's artifact-uploader component
+        name: area/prow/artifact-uploader
```

Comment:
Do we really need to be so specific? Is there a way we can force area/prow/artifact-uploader labels to have an area/prow label?

Reply:
Do we not? I would _love_ this when I am making tracking issues or trying to look at "what did we do for component X in this sprint/quarter" type of questions. I think they will get both labels from the auto-label OWNERS files, yes.

Reply:
yeah, we should still get the area/prow from the prow/owners, sgtm